woodwork/tests/accessor/test_table_accessor.py

```diff
@@ -1861,6 +1861,14 @@ def test_accessor_drop(sample_df):
     assert single_input_df.ww.schema == list_input_df.ww.schema
     assert to_pandas(single_input_df).equals(to_pandas(list_input_df))
 
+    if isinstance(sample_df, pd.DataFrame):
```

Comment:
since the ValueError's being raised for dask and koalas aren't yet getting covered, it's probably worth adding those checks here or in a test that's just checking the errors

Reply:
Fixed!